internal/identity/oidc/azure/azure.go

```diff
@@ -8,15 +10,22 @@ import (
 	"net/url"
 	"time"
 
-	oidc "github.com/coreos/go-oidc"
+	go_oidc "github.com/coreos/go-oidc"
 	"golang.org/x/oauth2"
 
 	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/identity"
+	"github.com/pomerium/pomerium/internal/identity/oidc"
 	"github.com/pomerium/pomerium/internal/log"
 	"github.com/pomerium/pomerium/internal/sessions"
 	"github.com/pomerium/pomerium/internal/version"
 )
 
+var _ identity.Authenticator = &AzureProvider{}
```

Comment:
@desimone Please be kind to help me understand why we need to do this.

Reply:
Ah, for sure. This is a trick to force compile time assertions that an interface is satisfied. In particular, that AzureProvider implements Authenticator. 

https://commaok.xyz/post/compile-time-assertions/

Reply:
Thanks.